skrf/vectorFitting.py

```diff
@@ -1388,7 +1368,7 @@ def get_model_response(self, i: int, j: int, freqs: Any = None) -> np.ndarray:
                 resp += residues[i] / (s - pole) + np.conjugate(residues[i]) / (s - np.conjugate(pole))
         return resp
 
-    @check_plotting
+    @check_plotting(mplt)
```

Comment:
We could extend the decorator, so that mplt is automatically passed as kwargs for the decorated function. We would not be required to pass the matplotlib instance every time.

Reply:
I am not sure how to do this.